### DISCUSSION TITLE
Satisfactory Savegame

### DISCUSSION THREAD

**Discussion by Eagle0112MK** [GameServerManagers/LinuxGSM#4247] [Q&A]
Hi Community,

can someone tell me where the savegame is created by satisfactory on a Ubunto Linux server because the savegame folder is empty but a savegame is created which you can also load

**Comment by EuropaSteve** [comment]
/home/sfserver/.config/Epic/FactoryGame/Saved/SaveGames/server
.config is a hidden file. If you don't see it you'll have to un-hide in your FTP program.  If that's what your using.
